requirements-test.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements.txt
 coverage==6.3.2
+django-coverage-plugin==2.0.2
```

Comment:
It seems that it is mostly related with Django templates but where we use those in the `safe-transaction-service`?

Reply:
I will use it either way to update configuration